ksp_plugin_adapter/localization/zh-cn.cfg

```diff
@@ -186,7 +188,7 @@ Localization {
     #Principia_FlightPlan_StatusMessage_ChangeFlightPlan = 请更换轨道规划
     #Principia_FlightPlan_StatusMessage_FailedError = 计算失败，出现错误 <<1>> 错误信息 \"<<2>>\"  // <<1>> status_.error <<2>> status_.message
     #Principia_FlightPlan_StatusMessage_TimeOut = 在<<1>>之后  // <<1>> Timeout
```

Comment:
Change this to <<1>> (it is weird that it is its own string anyway), we’ll deal with wrapping it below (and I might just get rid of it later on).

Reply:
As in, just `#Principia_FlightPlan_StatusMessage_FailedError = <<1>>`?

Reply:
Wrong line, but
```
#Principia_FlightPlan_StatusMessage_TimeOut = <<1>>
```
(That is what goes into the <<1>> of the other two strings below)

Reply:
Ah. That  `<<1>>` I figured, thought you were referring to something else for whatever reason.